Client/Ecosia/L10N/String.swift

```diff
@@ -182,7 +182,18 @@ extension String {
         case ecosiaLogoAccessibilityLabel = "Ecosia logo"
         case done = "Done"
         case findInPage = "Find in page"
+        case pageControlDots = "Page control dots"
+        case backButtonAccessibilityLabel = "Back button"
```

Comment:
I believe that an `UIButton` by default will be voiced as accessibility label + "button", so is the word button here needed? Even if it is not an `UIButton` you can also make it work as such by setting `.accessibilityTraits = .button`. I guess this might work best with how the user is used to accessibility in other apps.